DESCRIPTION

```diff
@@ -0,0 +1,75 @@
+Package: remindmodel
```

Comment:
REMIND is not a package.

Reply:
That's true. I got the idea to create a DESCRIPTION anyways from [Hadley Wickham](https://r-pkgs.org/Metadata.html), all the way at the bottom he describes this off-label use of a DESCRIPTION. Tools like renv and devtools can work with this DESCRIPTION even if the whole thing is not a package. Apparently the way you set a project's name in a DESCRIPTION is via the `Package` field, in the `Description` field it is clearly state that this is not a package though.

Reply:
I'm fine with adding the DESCRIPTION file. Will it then complement the automatic installation of needed packages and dependencies by `renv`?

Reply:
Yes, `renv::dependencies` will check the DESCRIPTION in addition to scanning all R and Rmd files to determine dependencies, and this is what's installed when running R with an empty renv.

Reply:
> I got the idea to create a DESCRIPTION anyways from [Hadley Wickham](https://r-pkgs.org/Metadata.html)

… from Hadley Wickham's book on writing R packages, which REMIND isn't.

`renc::dependencies()` also parses DESCRIPTION files without a `Package` field.

Reply:
I removed the `Package: remindmodel` line